### PR TITLE
Added new logic for tiles' color generation

### PR DESCRIPTION
**Pull request by tejas-k3**
* Now set of tiles will have themed colors
* Fixed console log issue wrt manifest file

### CODE REVIEW

**Review by siddhi-gh — COMMENTED** [review]
@tejas-k3 You should attach screenshots of any visual changes that have been made to the App in this PR in the PR description. Precisely indicating the before and after state of the elements.
It gives the reviewer a better idea to what to specifically test while reviewing the PR and code changes.

File: src/utils/getRandomColors.js [hunk 1]
```diff
@@ -1,8 +1,69 @@
...
+const getOriginalColor = () => {
   const letters = "0123456789ABCDEF";
   let color = "#";
   for (let i = 0; i < 6; i++) {
     color += letters[Math.floor(Math.random() * 16)];
   }
   return color;
 };
+
+const getColorHue = (color) => {
```
Comment (siddhi-gh) [inline src/utils/getRandomColors.js:38]: This new utility function should be added to `src/utils/` path.

File: package.json [hunk 2]
```diff
@@ -18,8 +18,8 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "export SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
-    "build": "export SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts build",
+    "start": "react-scripts start",
+    "build": "react-scripts build",
```
Comment (siddhi-gh) [inline package.json:22]: We need to figure this out.

**Review by siddhi-gh — COMMENTED** [review]
@tejas-k3 The new color scheme looks good.

https://github.com/tejas-k3/tejas-kothari-portfolio/assets/91131867/f5ce35f8-25ec-4bff-8f4f-50a19c1ab985

 Do check my review comments.

File: package.json (hunk 2 above)
Comment (tejas-k3) [reply package.json:22]: Will create a ticket & address it separately

**Review by siddhi-gh — APPROVED** [review]
Thanks @tejas-k3, changes look good. Approved.

**Pull request merged**